src/AI_ExtensibleSaveFormat/AI.ExtendedSave.Hooks.cs

```diff
@@ -14,11 +22,248 @@ internal static partial class Hooks
             [HarmonyPatch(typeof(CustomClothesFileInfoAssist), nameof(CustomClothesFileInfoAssist.CreateClothesFileInfoList))]
             [HarmonyPatch(typeof(GameCoordinateFileInfoAssist), nameof(GameCoordinateFileInfoAssist.CreateCoordinateFileInfoList))]
             private static void CreateListPrefix() => LoadEventsEnabled = false;
+
             [HarmonyPostfix]
             [HarmonyPatch(typeof(CustomCharaFileInfoAssist), nameof(CustomCharaFileInfoAssist.CreateCharaFileInfoList))]
             [HarmonyPatch(typeof(CustomClothesFileInfoAssist), nameof(CustomClothesFileInfoAssist.CreateClothesFileInfoList))]
             [HarmonyPatch(typeof(GameCoordinateFileInfoAssist), nameof(GameCoordinateFileInfoAssist.CreateCoordinateFileInfoList))]
             private static void CreateListPostfix() => LoadEventsEnabled = true;
+
+
+            public static void InstallMainGameHooks(Harmony harmony)
+            {
+                // Minimizing the interference between games to limiting the harmony hook initialization
+                // MainGame Save/Load - PostFix
+                harmony.Patch(
+                    typeof(SaveData).GetMethod("Load", new[] {typeof(BinaryReader)}),
+                    null, new HarmonyMethod(typeof(Hooks).GetMethod(nameof(OnLoadMainGame)))
+                );
+                harmony.Patch(
+                    typeof(SaveData).GetMethod("SaveFile", new[] {typeof(BinaryWriter)}),
+                    null, new HarmonyMethod(typeof(Hooks).GetMethod(nameof(OnSaveMainGame)))
+                );
+
+                // HousingData Save/Load - Transpiler
+                // Based on 2021 Version
+                harmony.Patch(
+                    typeof(CraftInfo).GetMethod("Load", new[] {typeof(string)}),
+                    null, null, new HarmonyMethod(typeof(Hooks).GetMethod(nameof(OnCraftLoad)))
+                );
+                harmony.Patch(
+                    typeof(CraftInfo).GetMethod("Save", new[] {typeof(string), typeof(byte[])}),
+                    null, null, new HarmonyMethod(typeof(Hooks).GetMethod(nameof(OnCraftSave)))
+                );
+            }
+
+            /// <summary>
+            ///
```

Comment:
Missing docs, either fill in or remove completely. There's no need to add docs in internal and private classes unless it's useful for other developers.

Reply:
noted.

Reply:
https://github.com/IllusionMods/BepisPlugins/pull/122/commits/a57e34b7c5f0c984a134ea1e0023d8c6bfee7d86